### DISCUSSION TITLE
Account broken after activating 2FA

### DISCUSSION THREAD

**Discussion by ghost** [community/community#73451] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Tittle says it all, my main account got "soft-locked" after activating it, I can't even visit it without being logged, I get a 404 error to everything related to my account, any deployment simply doesn't work either.

So much so that I tried making a ticket with my main account and it gives 404 too 
https://github.com/orgs/community/discussions/73450
Seriously, how is this even possible.

https://github.com/ongezell as you can see, 404 error, the profile is visible if I'm logged into the said account.

**Comment by Habibsultani** [comment]
Hi,  
I have just a few suggestion: 

1: If you're locked out of your account, you can try using the account recovery options provided by GitHub to regain access. This includes resetting your password or recovering your username.

2: To report account-specific issues, you can try contacting GitHub Support via their [contact page](https://support.github.com/contact). Since you mentioned that you can't even create a ticket from your main account, you might need to use a different GitHub account to reach out to support.

3: Before contacting support, you can also check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing issues with the platform that might be affecting your account.

**Reply by ghost** [reply]
I appreciate it but like I said, I'm not locked out of my account, I can login normally, is just that after activating the 2FA the account is basically unusable, I can't deploy anything, my profile is not visible to anyone neither are my repositories, it returns a 404 error.

**Reply by Habibsultani** [reply]
I recommend reaching out to GitHub's support team directly. They will have the necessary expertise and resources to assist you in resolving the problem with your account and restoring its functionality.
